Jenkinsfile

```diff
@@ -48,129 +44,103 @@ pipeline {
                     expression { env.CHANGE_TARGET != null }
                 }
             }
-            // TODO DDF-2971 refactor this stage from scripted syntax to declarative syntax to match the rest of the stages - https://issues.jenkins-ci.org/browse/JENKINS-41334
-            steps {
-                parallel(
-                    linux: {
-                        node('linux-large') {
-                            retry(3) {
```

Comment:
@mcalcote are we losing the retry functionality by switching away from the scripted syntax? Is there a way we can advise this still?

Reply:
Adding the retry logic back into the top level agent as well as each stage where we use an alternate agent.